src/fstests/fstests.jinja2

```diff
@@ -135,30 +142,23 @@ class Job(BaseJob):
         return node
 
     def _run(self, src_path):
-        fail_results = {
-            'node': {
-                'name': 'fstests',
-                'result': 'fail',
-                'state': 'done',
-            },
-            'child_nodes':[],
-        }
-        if not self._kernel_config(src_path):
-            return fail_results
-        if not self._kernel_build(src_path):
-            return fail_results
-        if not self._run_smoke_tests():
-            return fail_results
-        if not self._get_xml_from_vm(src_path):
-            return fail_results
+        results = self._generate_node_result('fstests', 'fail', 'done')
```

Comment:
Why are you setting fstests node result to `fail` in advance?
It should only be set when one of the child nodes is failing.

Reply:
@alexandrasp Any update on this?